kinto_wizard/yaml2kinto.py

```diff
@@ -84,5 +84,34 @@ def initialize_server(client, config):
                                                bucket=bucket_id,
                                                data=collection_data,
                                                permissions=collection_permissions)
+
+                # 2.2.1 For each collection, create its records.
+                collection_records = collection.get('records', {})
+                existing_records_ids = set()
+                if collection_exists:
+                    existing_records = client.get_records(bucket=bucket_id,
+                                                          collection=collection_id,
+                                                          **{"_fields": "id"})
+                    existing_records_ids = set([r["id"] for r in existing_records])
+                for record_id, record in collection_records.items():
+                    record_exists = record_id in existing_records_ids
+                    record_data = record.get('data', {})
+                    record_permissions = record.get('permissions', None)
+
+                    if not record_exists:
+                        batch.create_record(id=record_id,
```

Comment:
It depends what our understanding of the dump is.
If we want to put the content of the file in the database, then I guess we want to override the record with the one in the dump (in order to be able to remove field for instance)
If we want to add some info, then patch is a good fit.